doc/mgr/orchestrator.rst

```diff
@@ -256,9 +220,29 @@ the id is the numeric OSD ID, for MDS services it is the file system name::
     ceph orch daemon status <type> <instance-name> [--refresh]
 
 
+.. _orchestrator-cli-cephfs:
+    
+Depoying CephFS
+===============
+
+In order to set up a :term:`CephFS`, simply execute::
+
+    ceph fs volume create <fs_name> <placement spec>
+    
+Where
```

Comment:
```suggestion
Where:

Reply:
You could also do "Where..." but I'm not even sure this needs to be two separate dot points. I'd actually just recommend that you put it all together.

`Where ``name`` is the name of the CephFS, ``placement`` is a :ref:`orchestrator-cli-placement-spec`.`